tests/nnstreamer_filter_mxnet/Predictor.hh

```diff
@@ -155,7 +155,7 @@ Predictor::Predictor (const std::string &model_json_file, const std::string &mod
   }
   // Load the model
   LoadModel (model_json_file);
-  // Initilize the parameters
+  // Initialize the parameters
```

Comment:
suggestion : line-comment rule
I suggest fixing line-comment rule of this file (// -> /* */)

Reply:
Good, Updated!